app/javascript/controllers/query_string_controller.js

```diff
@@ -0,0 +1,23 @@
+import ApplicationController from './application_controller'
+
+/* This is the custom StimulusReflex controller for QueryStringReflex.
+ * Learn more at: https://docs.stimulusreflex.com
+ */
+export default class extends ApplicationController {
+  filter () {
+    this.stimulate('QueryStringReflex#filter')
+  }
+
+  afterReflex (element, reflex, error) {
+    if (!error) {
+      const params = new URLSearchParams(window.location.search.slice(1))
+      params.set('category', element.value)
```

Comment:
`Object.keys(Object.assign({}, this.element.dataset)).filter(attr => attr.startsWith(this.identifier)).forEach(attr => { this.element.dataset[attr] <- do cool stuff here })`

Reply:
I thnk we're slowly getting there...